ui/analyse/src/forecast/forecastView.ts

```diff
@@ -27,15 +27,18 @@ function onMyTurn(ctrl: AnalyseCtrl, fctrl: ForecastCtrl, cNodes: ForecastStep[]
   ]);
 }
 
+const nodeToForecastStep = (node: Tree.Node): ForecastStep => ({
+  ply: node.ply,
+  fen: node.fen,
+  uci: node.uci!,
+  san: node.san!,
+  check: node.check,
+  children: node.children.map(nodeToForecastStep),
```

Comment:
this field is never used, unfortunate that I had to do this
maybe there's a better way